flang/test/Driver/xlinker.f90

```diff
@@ -0,0 +1,8 @@
+! RUN: %flang -### -o /dev/null %s -Xlinker -rpath=/not/a/real/path 2>&1 | FileCheck --check-prefix=SINGLE %s
+! RUN: %flang -### -o /dev/null %s -Xlinker -rpath -Xlinker /not/a/real/path 2>&1 | FileCheck --check-prefix=MULTIPLE %s
+
+
+! SINGLE: "-rpath=/not/a/real/path"
+! MULTIPLE: "-rpath" "/not/a/real/path"
```

Comment:
nit: The problem with this is it does not check that the -Xlinker flag is actually passed to the linker

Reply:
Yes. I realized that soon immediately after I pushed up the changes (sigh). Thanks for taking multiple looks.